test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1

```diff
@@ -13,10 +13,15 @@ Describe "Get-Process for admin" -Tags @('CI', 'RequireAdminOnWindows') {
         }
     }
 
-    It "Should support -FileVersionInfo" -Skip:(!$IsWindows) {
```

Comment:
This test is wrong for non-Windows.  If you run the `Get-Process` line below on non-Windows, you'll see that FileVersion is `$null`.  "anything" -match $null returns $true.  This test for correct FileVersion being returned will only work on Windows.  Perhaps we can modify this test to work on both Windows and non-Windows:

```powershell
It "Should support -FileVersionInfo" {
  $pwshVersion = ...
  if ($IsWindows) {
     $pwshVersion | Should -BeExactly $PSVersionTable.PSVersion
 } else {
     $pwshVersion | Should -BeNullOrEmpty
 }
}
```

Reply:
Do we know that process' FileVersion always would be null or non windows platform?

Reply:
Yes, the FileVersion information is stored as resources in PE file format on Windows whereas Linux uses ELF format which is not compatible with this extended file information.